Content.Server/_DV/CosmicCult/MonumentSystem.cs

```diff
@@ -0,0 +1,369 @@
+using System.Linq;
+using Content.Server._DV.CosmicCult.Components;
+using Content.Server._DV.CosmicCult.EntitySystems;
+using Content.Server.Actions;
+using Content.Server.Atmos.Components;
+using Content.Server.Audio;
+using Content.Server.Chat.Systems;
+using Content.Server.Objectives.Components;
+using Content.Shared._DV.CCVars;
+using Content.Shared._DV.CosmicCult;
+using Content.Shared._DV.CosmicCult.Components;
+using Content.Shared._DV.CosmicCult.Prototypes;
+using Content.Shared.Audio;
+using Content.Shared.Damage;
+using Content.Shared.Interaction;
+using Content.Shared.Popups;
+using Content.Shared.Stacks;
+using Content.Shared.Temperature.Components;
+using Content.Shared.UserInterface;
+using Robust.Server.GameObjects;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Configuration;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Timing;
+
+namespace Content.Server._DV.CosmicCult;
+
+public sealed class MonumentSystem : SharedMonumentSystem
+{
+    [Dependency] private readonly ActionsSystem _actions = default!;
+    [Dependency] private readonly AppearanceSystem _appearance = default!;
+    [Dependency] private readonly ChatSystem _chatSystem = default!;
+    [Dependency] private readonly CosmicCorruptingSystem _corrupting = default!;
+    [Dependency] private readonly CosmicCultRuleSystem _cosmicRule = default!;
+    [Dependency] private readonly DamageableSystem _damage = default!;
+    [Dependency] private readonly DamageableSystem _damageable = default!;
+    [Dependency] private readonly EntityLookupSystem _lookup = default!;
+    [Dependency] private readonly IConfigurationManager _config = default!;
+    [Dependency] private readonly IGameTiming _timing = default!;
+    [Dependency] private readonly IPrototypeManager _protoMan = default!;
+    [Dependency] private readonly ServerGlobalSoundSystem _sound = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly SharedPopupSystem _popup = default!;
+    [Dependency] private readonly SharedUserInterfaceSystem _ui = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<MonumentComponent, InteractUsingEvent>(OnInfuseHeldEntropy);
+        SubscribeLocalEvent<MonumentComponent, ActivateInWorldEvent>(OnInfuseEntropy);
+    }
+
+
+    public override void Update(float frameTime) // This Update() can fit so much functionality in it
```

Comment:
Remove extra blank line
```suggestion
    }

    public override void Update(float frameTime)
```